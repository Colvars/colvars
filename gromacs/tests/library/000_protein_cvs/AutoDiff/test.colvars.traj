# step         alpha                  dihedPC              
           0    8.36517512054205e-01  -2.12766834807868e+00
           1    8.36405766419679e-01  -2.12968264113607e+00
           2    8.36303205720121e-01  -2.13167960318568e+00
           3    8.36202390746981e-01  -2.13367244075261e+00
           4    8.36092498129416e-01  -2.13570932765782e+00
           5    8.35961382156574e-01  -2.13784366106170e+00
           6    8.35795704781796e-01  -2.14011033033341e+00
           7    8.35588169487745e-01  -2.14250224138509e+00
           8    8.35341240201779e-01  -2.14495118133642e+00
           9    8.35075646652295e-01  -2.14731571085274e+00
          10    8.34827777768729e-01  -2.14938810487336e+00
          11    8.34642161350716e-01  -2.15090786644107e+00
          12    8.34556653334053e-01  -2.15159686737947e+00
          13    8.34590388102958e-01  -2.15119434312355e+00
          14    8.34737327530330e-01  -2.14950028143585e+00
          15    8.34967741024785e-01  -2.14640394254280e+00
          16    8.35236233864300e-01  -2.14191002556526e+00
          17    8.35490078600560e-01  -2.13614501779765e+00
          18    8.35676693813091e-01  -2.12934033057215e+00
          19    8.35752538813196e-01  -2.12182504079626e+00
          20    8.35682077723043e-01  -2.11396690205205e+00
