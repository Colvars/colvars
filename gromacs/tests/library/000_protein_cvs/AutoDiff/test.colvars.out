colvars: ----------------------------------------------------------------------
colvars: Please cite Fiorin et al, Mol Phys 2013:
colvars:   https://doi.org/10.1080/00268976.2013.813594
colvars: as well as all other papers listed below for individual features used.
colvars: This version was built with the C++11 standard or higher.
colvars: Summary of compile-time features available in this build:
colvars:   - SMP parallelism: enabled (num. threads = 2)
colvars:   - Multiple replicas: not available
colvars:   - Lepton custom functions: available
colvars:   - Tcl interpreter: not available
colvars: ----------------------------------------------------------------------
colvars: Start colvars Initialization.
colvars: ----------------------------------------------------------------------
colvars: Reading new configuration:
colvars: # units = "" [default]
colvars: # indexFile = "index.ndx"
colvars: The following index groups are currently defined:
colvars:   Protein (104 atoms)
colvars:   Protein_noH (51 atoms)
colvars:   Protein_Backbone (40 atoms)
colvars:   Protein_C-alpha (10 atoms)
colvars:   RMSD_atoms (10 atoms)
colvars:   Protein_C-alpha_1_2 (2 atoms)
colvars:   Protein_C-alpha_9_10 (2 atoms)
colvars:   Protein_C-alpha_1 (1 atoms)
colvars:   group1 (4 atoms)
colvars:   Protein_C-alpha_2 (1 atoms)
colvars:   group2 (4 atoms)
colvars:   Protein_C-alpha_3 (1 atoms)
colvars:   group3 (4 atoms)
colvars:   Protein_C-alpha_4 (1 atoms)
colvars:   group4 (4 atoms)
colvars:   Protein_C-alpha_5 (1 atoms)
colvars:   group5 (4 atoms)
colvars:   Protein_C-alpha_6 (1 atoms)
colvars:   group6 (4 atoms)
colvars:   Protein_C-alpha_7 (1 atoms)
colvars:   group7 (4 atoms)
colvars:   Protein_C-alpha_8 (1 atoms)
colvars:   group8 (4 atoms)
colvars:   Protein_C-alpha_9 (1 atoms)
colvars:   group9 (4 atoms)
colvars:   Protein_C-alpha_10 (1 atoms)
colvars:   group10 (4 atoms)
colvars:   heavy_atoms (51 atoms)
colvars:   prot_N (10 atoms)
colvars:   prot_CA (10 atoms)
colvars:   prot_C (10 atoms)
colvars:   prot_O (10 atoms)
colvars: # smp = on [default]
colvars: # colvarsTrajFrequency = 1
colvars: # colvarsRestartFrequency = 10
colvars: # scriptedColvarForces = off [default]
colvars: # scriptingAfterBiases = off [default]
colvars: # defaultInputStateFile = "" [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "alpha"
colvars:   Initializing a new "alpha" component.
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:     # prefix = "prot_"
colvars:     # hBondCoeff = 0.5 [default]
colvars:     # angleRef = 88 [default]
colvars:     # angleTol = 15 [default]
colvars:     # hBondCutoff = 0.33 [default]
colvars:     # hBondExpNumer = 6 [default]
colvars:     # hBondExpDenom = 8 [default]
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 1 [default]
colvars:   # lowerBoundary = 0 [default]
colvars:   # upperBoundary = 1 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = off [default]
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars:   Initializing a new collective variable.
colvars:   # name = "dihedPC"
colvars:   Initializing a new "dihedralPC" component.
colvars:   Warning: cannot cite unknown feature "dihedPC colvar component"
colvars:     # name = "" [default]
colvars:     # componentCoeff = 1 [default]
colvars:     # componentExp = 1 [default]
colvars:     # period = 0 [default]
colvars:     # wrapAround = 0 [default]
colvars:     # forceNoPBC = off [default]
colvars:     # scalable = on [default]
colvars:     # prefix = "prot_"
colvars:     # vectorFile = "../Common/eigenvectors-localmin"
colvars:     # vectorNumber = 2
colvars:   All components initialized.
colvars:   # timeStepFactor = 1 [default]
colvars:   # width = 1 [default]
colvars:   # lowerBoundary = 0 [default]
colvars:   # upperBoundary = 1 [default]
colvars:   # hardLowerBoundary = off [default]
colvars:   # hardUpperBoundary = off [default]
colvars:   # expandBoundaries = off [default]
colvars:   # extendedLagrangian = off [default]
colvars:   # outputValue = on [default]
colvars:   # outputVelocity = off [default]
colvars:   # outputTotalForce = off [default]
colvars:   # outputAppliedForce = off [default]
colvars:   # subtractAppliedForce = off [default]
colvars:   # runAve = off [default]
colvars:   # corrFunc = off [default]
colvars: ----------------------------------------------------------------------
colvars: Collective variables initialized, 2 in total.
colvars: No collective variables biases were defined.
colvars: ----------------------------------------------------------------------
colvars: Collective variables module (re)initialized.
colvars: ----------------------------------------------------------------------
colvars: Current simulation parameters: initial step = 0, integration timestep = 1
colvars: Updating atomic parameters (masses, charges, etc).
colvars: Re-initialized atom group for variable "alpha":0/0. 1 atoms: total mass = 12.011, total charge = -0.1.
colvars: Re-initialized atom group for variable "alpha":0/1. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "alpha":0/2. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "alpha":0/3. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "alpha":0/4. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "alpha":0/5. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "alpha":0/6. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "alpha":0/7. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "alpha":0/8. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "alpha":0/9. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "alpha":0/10. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "alpha":0/11. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "alpha":0/12. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "alpha":0/13. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "alpha":0/14. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "alpha":0/15. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "alpha":0/16. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "alpha":0/17. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "alpha":0/18. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "alpha":0/19. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "alpha":0/20. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "alpha":0/21. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "alpha":0/22. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "alpha":0/23. 1 atoms: total mass = 1.008, total charge = 0.09.
colvars: Re-initialized atom group for variable "alpha":0/24. 2 atoms: total mass = 30.006, total charge = -0.98.
colvars: Re-initialized atom group for variable "alpha":0/25. 2 atoms: total mass = 30.006, total charge = -0.98.
colvars: Re-initialized atom group for variable "alpha":0/26. 2 atoms: total mass = 30.006, total charge = -0.98.
colvars: Re-initialized atom group for variable "alpha":0/27. 2 atoms: total mass = 30.006, total charge = -0.98.
colvars: Re-initialized atom group for variable "alpha":0/28. 2 atoms: total mass = 30.006, total charge = -0.98.
colvars: Re-initialized atom group for variable "alpha":0/29. 2 atoms: total mass = 17.007, total charge = -0.42.
colvars: Re-initialized atom group for variable "dihedPC":0/0. 1 atoms: total mass = 14.007, total charge = -0.62.
colvars: Re-initialized atom group for variable "dihedPC":0/1. 1 atoms: total mass = 12.011, total charge = -0.1.
colvars: Re-initialized atom group for variable "dihedPC":0/2. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "dihedPC":0/3. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "dihedPC":0/4. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "dihedPC":0/5. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "dihedPC":0/6. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "dihedPC":0/7. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "dihedPC":0/8. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "dihedPC":0/9. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "dihedPC":0/10. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "dihedPC":0/11. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "dihedPC":0/12. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "dihedPC":0/13. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "dihedPC":0/14. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "dihedPC":0/15. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "dihedPC":0/16. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "dihedPC":0/17. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "dihedPC":0/18. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "dihedPC":0/19. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "dihedPC":0/20. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "dihedPC":0/21. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "dihedPC":0/22. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "dihedPC":0/23. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "dihedPC":0/24. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "dihedPC":0/25. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "dihedPC":0/26. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "dihedPC":0/27. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "dihedPC":0/28. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "dihedPC":0/29. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "dihedPC":0/30. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "dihedPC":0/31. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "dihedPC":0/32. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "dihedPC":0/33. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "dihedPC":0/34. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "dihedPC":0/35. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "dihedPC":0/36. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "dihedPC":0/37. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "dihedPC":0/38. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "dihedPC":0/39. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "dihedPC":0/40. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "dihedPC":0/41. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "dihedPC":0/42. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "dihedPC":0/43. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "dihedPC":0/44. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "dihedPC":0/45. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "dihedPC":0/46. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "dihedPC":0/47. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "dihedPC":0/48. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "dihedPC":0/49. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "dihedPC":0/50. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "dihedPC":0/51. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "dihedPC":0/52. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "dihedPC":0/53. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "dihedPC":0/54. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "dihedPC":0/55. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "dihedPC":0/56. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "dihedPC":0/57. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "dihedPC":0/58. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "dihedPC":0/59. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "dihedPC":0/60. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "dihedPC":0/61. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "dihedPC":0/62. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "dihedPC":0/63. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "dihedPC":0/64. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: Re-initialized atom group for variable "dihedPC":0/65. 1 atoms: total mass = 12.011, total charge = 0.07.
colvars: Re-initialized atom group for variable "dihedPC":0/66. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "dihedPC":0/67. 1 atoms: total mass = 1.008, total charge = 0.09.
colvars: Re-initialized atom group for variable "dihedPC":0/68. 1 atoms: total mass = 12.011, total charge = 0.51.
colvars: Re-initialized atom group for variable "dihedPC":0/69. 1 atoms: total mass = 1.008, total charge = 0.09.
colvars: Re-initialized atom group for variable "dihedPC":0/70. 1 atoms: total mass = 1.008, total charge = 0.09.
colvars: Re-initialized atom group for variable "dihedPC":0/71. 1 atoms: total mass = 14.007, total charge = -0.47.
colvars: 
colvars: SUMMARY OF COLVARS FEATURES USED SO FAR AND THEIR CITATIONS:
colvars: 
colvars: - GROMACS engine:
colvars:   Abraham2015 https://doi.org/10.1016/j.softx.2015.06.001
colvars: 
colvars: - Colvars module:
colvars: - Optimal rotation via flexible fitting:
colvars: - alpha colvar component:
colvars: - angle colvar component:
colvars: - dihedral colvar component:
colvars: - hBond colvar component:
colvars:   Fiorin2013 https://doi.org/10.1080/00268976.2013.813594
colvars: 
colvars: - Colvars-GROMACS interface:
colvars:   n/a 
colvars: 
colvars: The final output state file will be "test.colvars.state".
colvars: ----------------------------------------------------------------------
colvars: End colvars Initialization.
colvars: 
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.colvars.state".
colvars: Synchronizing (emptying the buffer of) trajectory file "test.colvars.traj".
colvars: Saving collective variables state to "test.colvars.state".
colvars: Saving collective variables state to "test.colvars.state".
